src/Calculator/Resources/he-IL/Resources.resw

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looks like there's still some change. (Maybe it's end-of-line. Could you revert it back?